airbase/pom.xml

```diff
@@ -1130,21 +1132,29 @@
 
             <!-- Java APIs -->
             <dependency>
-                <groupId>javax.inject</groupId>
-                <artifactId>javax.inject</artifactId>
-                <version>${dep.javax-inject.version}</version>
+                <groupId>jakarta.inject</groupId>
+                <artifactId>jakarta.inject-api</artifactId>
+                <version>${dep.jakarta-inject.version}</version>
             </dependency>
+
+            <!-- servlet -->
             <dependency>
-                <groupId>javax.validation</groupId>
-                <artifactId>validation-api</artifactId>
-                <version>${dep.javax-validation.version}</version>
+                <groupId>jakarta.servlet</groupId>
```

Comment:
Per the Jetty team, we want to use `jetty-jakarta-servlet-api`. When 6.0 is out, we can switch to the official `jakarta.servlet` dependency.